templates/blog/post_detail.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>{{ post.title }}</title>
+{% endblock %}
+
+{% block content %}
+  <h2>{{ post.title }}</h2>
+  <div>
+    <div> Author: {{ post.owner }} Created: {{ post.created_time }}</div>
+    <div>{{ post.content }}</div>
+    <br>
+    <b>Comments</b>
+    <ul class="list-group">
+      {% for comment in post.commentary.all %}
+        <li class="list-group-item">
+          <div>
+            <strong>{{ comment.user }}</strong> - <small>{{ comment.created_time }}</small>
+            <hr>
+            {{ comment.content }}
+          </div>
+        </li>
+      {% empty %}
+        <li>No comments yet.</li>
+      {% endfor %}
+    </ul>
+
+    <h3>Add a Comment</h3>
+    {% load crispy_forms_tags %}
```

Comment:
The `{% load crispy_forms_tags %}` is usually placed at the top of the file, right after the `{% extends 'base.html' %}`. This helps in maintaining code consistency and readability.